shared/actions/config.js

```
@@ -225,12 +225,10 @@ function _loadAvatarHelper(action: AvatarHelperAction) {
 }
 
 function _loadAvatarHelperError(error: Error, action: AvatarHelperAction) {
-  // Ignore api errors
-  if (error.code === RPCTypes.constantsStatusCode.scapinetworkerror) {
-    throw error // TEMP
-  } else {
+  if (error.code === RPCTypes.constantsStatusCode.scinputerror) {
```

only show this type of error, all other errors are ignored